Library/Homebrew/rubocops/lines.rb

```diff
@@ -181,6 +170,11 @@ def unless_modifier?(node)
 
           node.modifier_form? && node.unless?
         end
+
+        # Finds `depends_on "foo" if build.with?("bar")` or `depends_on "foo" if build.without?("bar")`
+        def_node_search :depends_on_build_with, <<~EOS
+          (if $(send (send nil? :build) {:with? :without?} str) (send nil? :depends_on str) nil?)
```

Comment:
Does Rubocop’s node hook syntax let you add a line break and an indentation between the conditional expression and the statement? Or would that affect the semantics?

Like we do it for `conditional_dependencies`:
https://github.com/Homebrew/brew/blob/bc4a10804586c8f7a55505923dc1c627b66735d4/Library/Homebrew/rubocops/lines.rb#L495-L496

Reply:
Good idea. That will definitely make readability easier.